README.md

```diff
@@ -55,7 +54,20 @@ module.exports = {
 ```
 
 This will enable CSS optimization only in production mode.
-If you want to run it also in development set the `optimization.minimize` option to `true`.
+
+If you want to run it also in development set the `optimization.minimize` option to `true`:
+
+**webpack.config.js**
+
+```js
+[...]
```

Comment:
Avoid non JS code in code examples, use comments

Reply:
Fixed